### DISCUSSION TITLE
Showing progress value inside a FluentProgressRing

### DISCUSSION THREAD

**Discussion by giancarlomelis** [microsoft/fluentui-blazor#1668] [Q&A] [answered]
I'm trying to understand how to insert the progress value into a FluentProgressRing component, as happens for example when loading the fluentui-blazor.net demo website:

![image](https://github.com/microsoft/fluentui-blazor/assets/18438837/24fce418-9db5-4027-be53-1148bf369fb2)

However, it seems that the ChildContent of the FluentProgressRing is never shown.

**Comment by vnbaaij** [comment, marked as answer]
The animated ring when starting the demo site is just (a slightly adapted version of) the standard thing that comes with the Blazor template.
The FluentProgressRing is not designed to show any text inside. See also the wb component version at https://fluentuipr.z22.web.core.windows.net/heads/master/web-components/storybook/index.html?path=/docs/components-progress-ring--progress-ring
